spaces/domains.json

```
@@ -157,5 +157,6 @@
   "governance.curio.cards": "curiovoters.eth",
   "vote.forkenomics.com": "forkenomics.eth",
   "snapshot.comp.vote": "comp-vote.eth",
-  "gov.tusker.finance": "tuskerfinance.eth"
+  "gov.tusker.finance": "tuskerfinance.eth",
+  "gov.sunicoin.org": "suni.eth
```

You will miss a quote at the end of your line. Please update your file before commit again cause some new domains have been added.
```suggestion
  "gov.sunicoin.org": "suni.eth"
```